### PR TITLE
Sc 4162 branch support

### PR DESCRIPTION
**Pull request by saraiva132**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ USAGE:
 
 OPTIONS:
     -c, --cache-directory <CACHE_DIRECTORY>
-            location of the protofetch cache directory [default: .protofetch_cache]
+            location of the protofetch cache directory [default: cache]
```
Comment (zhrebicek) [inline README.md:15]: Should not it be something like `~/.protofetch` ?

File: src/cli/args.rs [hunk 2]
```diff
@@ -12,7 +12,7 @@ pub struct CliArgs {
     #[clap(short, long, default_value = "protofetch.lock")]
     /// location of the protofetch lock file
     pub lockfile_location: String,
-    #[clap(short, long, default_value = ".protofetch_cache")]
+    #[clap(short, long, default_value = "cache")]
```
Comment (zhrebicek) [inline src/cli/args.rs:15]: Same as the readme, do now we want to have `~/.protofetch` ?

File: src/main.rs [hunk 3]
```diff
@@ -21,16 +21,19 @@ fn main() {
 
 fn run() -> Result<(), Box<dyn Error>> {
     let cli_args: CliArgs = CliArgs::parse();
-
-    let cache = ProtofetchCache::new(PathBuf::from(&cli_args.cache_directory))?;
+    let home_dir = home::home_dir().expect("Could not find home dir. Please define $HOME!");
```
Comment (zhrebicek) [inline src/main.rs:24]: This should be reflected in the `cache_directory` option, or maybe the option should be dropped altogether when it is always in home.

File: README.md (hunk 1 above)
Comment (saraiva132) [reply README.md:15]: That is just the cache directly file within `~/.protofetch` maybe I should have left it as `./protofetch/cache` tbh

**Review by zhrebicek — APPROVED** [review]

**Pull request merged**